spec/openapi.yaml

```diff
@@ -120,6 +120,39 @@ paths:
         '500':
           description: An unexpected error occured.
 
+  /tag/metadata/delete:
+    post:
+      operationId: tag-metadata-delete
+      summary: Permanently deletes the specified metadata from an existing tag.
+      description: >
+        Provide the metadata names that you want to delete.
+      tags:
+        - Tags
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/TagMetadataDeleteRequest'
+      responses:
+        '200':
+          description: Tag metadata successfully deleted.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/TagMetadataDeleteResponse'
+        '204':
+          description: >
+            (No Content) metadata was not found or already deleted.
```

Comment:
```suggestion
```

@chanddan I didn't realise that we were deleting multiple tag metadata entries in one request. If that is the case then it seems simpler and less confusing to always return 200 OK when tag was found and there was no error processing the request, whether metadata was deleted or not. My 204 suggestion doesn't seem all that great now. What do you think?

Reply:
I have added a reply in `connect-api-server/pull-requests/159/overview`